src/Conf.ml

```diff
@@ -726,6 +763,14 @@ let inputs =
   mk ~default
     Arg.(value & pos_all file_or_dash default & info [] ~doc ~docv ~docs)
 
+let inside_project_only =
+  let doc =
+    "Do not read .ocamlformat config files outside the project. If no \
+     config file is found, disable ocamlformat."
```

Comment:
What is considered to be inside a project should be documented explicitly. Maybe: `The project root of an input file is taken to be the nearest ancestor directory that contains .git, .hg, or dune-project.`

Reply:
updated